item-spec/json-schema/item.json

```diff
@@ -204,21 +204,6 @@
               "title": "Link title",
               "type": "string"
             }
-          },
```

Comment:
The removal is likely unintentional?

Reply:
I think this is an error when merging with dev branch